tests/astrometry_net_data/photocal/andConfig.py

```diff
@@ -0,0 +1,7 @@
+root.defaultMagColumn = 'r'
+root.starGalaxyColumn = 'star'
+#root.variableColumn = 'variable'
```

Comment:
I am happy to remove uncommented code, but I don't think it is safe to remove the configs without extensive testing, and the example you give above looks useful.